### ISSUE TITLE
Syntax error in delete predicate can cause all data in range to be deleted

### ISSUE THREAD

**Issue by danxmoran** [influxdata/influxdb#20796]
Originally reported in the community [here](https://community.influxdata.com/t/influx-delete-bad-behaviour-leads-to-data-loss/18505).

> Running influx delete with only a small syntax error in the --predicate flag deletes ALL data with timestamps between the specified --start and --stop times in the specified bucket.
This way I lost all the data in my bucket when I wanted to delete only certain data!

Still waiting on the bad predicate so we can reproduce.

**Comment by danxmoran** [comment]
Duplicate of #20825

**Comment by danxmoran** [comment]
Closing this in favor of the user-reported issue
